### PR TITLE
Disable api permission in debug mode

### PR DESCRIPTION
**Pull request by xiangge**
Disable api permission when DEBUG is set to True.
Modify the permission part in development.rst and settings_local.py.dist.

### CODE REVIEW

File: pdc/apps/auth/permissions.py [hunk 1]
```diff
@@ -20,7 +20,10 @@ def has_permission(self, permission, request, view):
         internal_permission = self._convert_permission(request.method)
         if not internal_permission or (read_permission_for_all() and internal_permission == 'read'):
             return True
-        return self._has_permission(internal_permission, request.user, str(view.__class__), api_name)
+        if settings.DEBUG:
+            return True
+        else:
```
Comment (hluk) [inline pdc/apps/auth/permissions.py]: Else branch not needed.

File: pdc/settings_local.py.dist [hunk 2]
```diff
@@ -22,41 +22,16 @@
...
-#         'DEFAULT_PAGINATION_CLASS': 'pdc.apps.common.pagination.AutoDetectedPageNumberPagination',
-#     }
 
 
 import os.path
 
 BASE_DIR = os.path.dirname(os.path.dirname(__file__))
 
+# NOTE: when you run PDC locally, you may not want to enable
+# the permission, just need to set DEBUG to True
```
Comment (hluk) [inline pdc/settings_local.py.dist]: "permission" -> "permission checks"

File: pdc/apps/auth/permissions.py (hunk 1 above)
Comment (xiangge) [inline pdc/apps/auth/permissions.py]: Good idea, have changed that.

File: pdc/settings_local.py.dist (hunk 2 above)
Comment (xiangge) [inline pdc/settings_local.py.dist]: Yes, this is more clearly, changed. thx

**Comment by xiangge** [comment]
@simozhan yes, correct, I have removed the modified codes in permissions.py, just need to set "DISABLE_RESOURCE_PERMISSION_CHECK" with "True" can work.

**Comment by simozhan** [comment]
:+1:

**Comment by xiangge** [comment]
Close as patch has already merged.

**Pull request closed without merging**